views/dashboard/about-page/components/FormView.tsx

```diff
@@ -0,0 +1,201 @@
+import { use, useEffect } from 'react'
+import { useDispatch } from 'react-redux'
+import useForm from '@/utils/hooks/useForm'
+import FormValidations from '@/utils/validations/forms'
+import { FileUpload, FormButtonGroup, TextField } from '@/components/form'
+import { updateAboutPageData } from '@/lib/db/cms/about-page'
+import { setAbout } from '@/lib/redux/slices/aboutPage'
+import { StyledForm } from './StyledFormView'
+import { handleCloudUpload } from '@/utils/fileConversions'
+import { enqueueSnackbar } from 'notistack'
+import { statuses, messages } from '@/utils/banners'
+
+const initialState = {
+    name: { value: '', error: '' },
+    description: { value: '', error: '' },
+    header: { value: '', error: '' },
+    subHeader: {
+        value: '',
+        error: ''
+    },
+    img: [
+        {
+            name: '',
+            src: '',
+            title: '',
+            type: ''
+        }
+    ],
+    role: {
+        value: '',
+        error: ''
+    },
+    title: {
+        value: '',
+        error: ''
+    }
+}
+
+const validations = {
+    name: FormValidations.yolo,
+    description: FormValidations.yolo,
+    header: FormValidations.yolo,
+    subHeader: FormValidations.yolo,
+    img: FormValidations.yolo,
+    role: FormValidations.yolo,
+    title: FormValidations.yolo
+}
+
+const FormView = ({ FormData }: any) => {
+    const dispatch = useDispatch()
+
+    const { description, header, subHeader, img, role, title, name, id } =
+        FormData
```

Comment:
FormData makes more sense to be called `store`. It's the data coming from the redux store. The form data is the data managed by the useForm hook.

Reply:
Bet and ill camelcase store